README.md

```diff
@@ -4,6 +4,13 @@ A container to create disk images from [bootc](https://github.com/containers/boo
 especially oriented towards [Fedora/CentOS bootc](https://docs.fedoraproject.org/en-US/bootc/) or
 derivatives.
 
+## Project
```

Comment:
I'm slightly confused, it seems we already have this in https://github.com/osbuild/bootc-image-builder/blob/main/README.md#-project (i.e. much further down in this file)

Reply:
Good point, I opened a dozen of these PRs and overlooked this. Deleted the section down, I think it is better at the top this is where all other projects have it.